app/controllers/subscriptions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SubscriptionsController < ApplicationController
+  def new
+    subscription = Subscription.new(search_criteria: search_criteria.to_json)
+    @subscription = SubscriptionPresenter.new(subscription)
+  end
+
+  def create
+    subscription = Subscription.new(subscription_params)
+    @subscription = SubscriptionPresenter.new(subscription)
+
+    render 'new' unless @subscription.valid?
```

Comment:
If it is valid, where does the user go?

Reply:
Nowhere on this PR 😈 

https://trello.com/c/naFQHYhF/659-123-email-alerts-subscription-confirmation addreses that.